client/lib/directly/test/index.js

```diff
@@ -96,22 +80,20 @@ describe( 'index', () => {
 		const name = 'Richie Rich';
 		const email = 'richie@richenterprises.biz';
 
-		it( 'does nothing if Directly hasn\'t been initialized', () => {
+		it( 'initializes Directly if it hasn\'t already been initialized', () => {
```

Comment:
Our eslint is set to: 
https://github.com/Automattic/wp-calypso/blob/master/.eslintrc.js#L52
```js
quotes: [ 1, 'single', 'avoid-escape' ],
```
http://eslint.org/docs/rules/quotes

So either single quotes here with an escape, or double quotes without escaping.